projects/sunbird-video-player/src/lib/sunbird-video-player.component.ts

```diff
@@ -103,6 +98,12 @@ export class SunbirdVideoPlayerComponent implements OnInit, AfterViewInit, OnDes
     this.sideMenuConfig = { ...this.sideMenuConfig, ...this.playerConfig.config.sideMenu };
     this.viewerService.initialize(this.playerConfig);
     this.videoPlayerService.initialize(this.playerConfig);
+    window.addEventListener("offline", (event) => {
```

Comment:
ngOnDestroy window.removeEventListener   move that function outside so that you can remove it